app/components/viral/flash_component.html.erb

```diff
@@ -2,69 +2,60 @@
   data-controller="viral--flash"
   data-viral--flash-timeout-value="<%= timeout %>"
   data-viral--flash-type-value="<%= type %>"
+  id="toast-<%= type %>-<%= object_id %>"
   class="
-    flex items-center w-full max-w-xs p-4 mb-4 text-slate-500 bg-white rounded-lg
-    shadow dark:text-slate-400 dark:bg-slate-800
+    flex items-center w-full max-w-xs p-4 mb-4 text-slate-700 bg-white rounded-lg
+    shadow-lg dark:text-slate-400 dark:bg-slate-800 transition-opacity duration-300
+    ease-out
   "
-  role="alert"
+  <% if type.to_s == 'error' %>
+    role="alert"
+    aria-live="assertive"
+  <% else %>
+    role="status"
+    aria-live="polite"
+  <% end %>
+  aria-atomic="true"
   data-turbo-permanent="true"
 >
-  <% case type.to_s %>
-  <% when "success" %>
-    <div
-      class="
-        inline-flex items-center justify-center shrink-0 w-8 h-8 text-green-500
-        bg-green-100 rounded-lg dark:bg-green-800 dark:text-green-200
-      "
-    >
-      <%= viral_icon(name: "check_circle_solid", classes: "w-5 h-5") %>
+  <div
+    class="
+      inline-flex items-center justify-center shrink-0 w-10 h-10 rounded-lg
+      <%= case type.to_s
```

Comment:
Can the setting of the classes be moved in the the component rb file?

Reply:
Updated in 3d5fb2fe97377aa1f314268fe7749259ef9681f9